lib/managers/groups.js

```diff
@@ -0,0 +1,321 @@
+};
+
+/**
+ * Used to fetch information about a group
+ *
+ * API Endpoint: '/groups/:groupID'
+ * Method: GET
+ *
+ * @param {string} groupID - The ID of the group to retrieve
+ * @param {?Object} qs - Additional options passed by query string, can be left null in most cases
+ * @param {Function} callback - Passed the group object if successful, error otherwise
+ * @returns {void}
+ */
+Groups.prototype.get = function(groupID, qs, callback) {
+
+	var apiPath = urlPath(BASE_PATH, groupID),
+		params = {
+			qs: qs
+		};
+
+	this.client.get(apiPath, params, this.client.defaultResponseHandler(callback));
+};
+
+/**
+ * Used to update or modify a group object
+ *
+ * API Endpoint: '/groups/:groupID'
+ * Method: PUT
+ *
+ * @param {string} groupID - The ID of the group to update
+ * @param {Object} options - Group fields to update
+ * @param {Function} callback - Passed the updated group object if successful, error otherwise
+ * @returns {void}
+ */
+Groups.prototype.update = function(groupID, options, callback) {
+
+	var apiPath = urlPath(BASE_PATH, groupID),
+		params = {
+			body: options
+		};
+
+	this.client.put(apiPath, params, this.client.defaultResponseHandler(callback));
+};
+
+/**
+ * Delete a group
+ *
+ * API Endpoint: '/groups/:groupID'
+ * Method: DELETE
+ *
+ * @param {string} groupID - The ID of the group to delete
+ * @param {Function} callback - Passed nothing if successful, error otherwise
+ * @returns {void}
+ */
+Groups.prototype.delete = function(groupID, callback) {
+
+	var apiPath = urlPath(BASE_PATH, groupID);
+
+	this.client.del(apiPath, null, this.client.defaultResponseHandler(callback));
+};
+
+/**
+ * Add a user to a group, which creates a membership record for the user
+ *
+ * API Endpoint: '/group_memberships'
+ * Method: POST
+ *
+ * @param {string} groupID - The ID of the group to add the user to
+ * @param {string} userID - The ID of the user to add the the group
+ * @param {?Object} options - Optional parameters for adding the user, can be left null in most cases
+ * @param {GroupUserRole} [options.role] - The role of the user in the group
+ * @param {Function} callback - Passed the membership record if successful, error otherwise
+ * @returns {void}
+ */
+Groups.prototype.addUser = function(groupID, userID, options, callback) {
+
+	var apiPath = urlPath(MEMBERSHIPS_PATH),
+		params = {
+			body: {
+				user: {
+					id: userID
+				},
+				group: {
+					id: groupID
+				}
+			}
+		};
+
+	Object.assign(params.body, options);
+
+	this.client.post(apiPath, params, this.client.defaultResponseHandler(callback));
+
+};
+
+/**
+ * Fetch a specific membership record, which shows that a given user is a member
+ * of some group.
+ *
+ * API Endpoint: '/group_memberships/:membershipID'
+ * Method: GET
+ *
+ * @param {string} membershipID - The ID of the membership to fetch
+ * @param {?Object} qs - Optional parameters, can be left null in most cases
```

Comment:
I realize the name is different because it's a get, but should we call all of these "options"?